OneLeaderFollowerSimulator en reporte.md

maux96 opened the pull request:
Revisar la info de reporte.md  (OneLeaderFollowerSimulator)

maux96 committed:
OneLeaderFollowerSimulator en reporte.md

Commit:
#9 Revisión

- Errores ortográficos.
- Mejoran2 el pseudocódigo.

maux96 committed:
Arreglo de reporte.md

MakeMake23 commented on docs/reporte.md:
```diff
@@ -24,33 +24,33 @@ En un sistema como este se pueden simular:
-  - $ n = n + 1 $ <!--@audit t falta la línea 94 d OneLeaderFollowersSimulator q va despue's d esta-->
-  - $ if~(n_1 == 1)~~~then~~~~(~t_{A_S}=genArrivalTimeOffset()<!--@audit este ) ta d +. También es mejor q lo hagas como en la conf, q cogen y dicen: generar t_{A_s} y... -->)~\wedge ~t_{A_2}=t + t_{A_S})$
+  - $ n = n + 1 $ 
+  - $ t_{A_1} = t + genArrivalToLidTimeOffset() $
```
mira, ak&iacute; lo q yo t dec&iacute;a a la hora d generar era q pusieras algo como esto:
![image](https://user-images.githubusercontent.com/62294636/142624116-28676ca3-63a4-496f-a193-497ee22cbe97.png)

como lo hacen en la conf, pa q c entienda mejor. Tonces eso lo haces cada vez q generes algo.

MakeMake23 commented on docs/reporte.md:
```diff
@@ -24,33 +24,33 @@ En un sistema como este se pueden simular:
 Incluyendo IA allá donde puede ser más útil :grin:.
 
 ## OneLeaderFollowersSimulator
```
Ah, mira, c&aacute;mbiale el t&iacute;tulo ese, ponle algo como _Modelo de un L&iacute;der y muchos Seguidores_.

MakeMake23 reviewed (changes requested):
Ya, mira los 2 comentarios q t deg

maux96 committed:
#9 Arreglo a reporte.md

-Se cambio el título del Modelo Líder y muchos seguidores.
-Se arreglaron detalles en el Modelo Líder y muchos seguidores.

MakeMake23 commented on docs/reporte.md:
```diff
@@ -49,19 +49,19 @@ Simulación de un l&iacute;der con un conjunto de servidores en paralelo.
-  - $ t_{A_1} = t + genArrivalToLidTimeOffset() $
-  - $ if~(n_1 == 1)~~~then~~~~t_{A_S}=genArrivalToFollTimeOffset()~\wedge ~t_{A_2}=t + t_{A_S})$   
+  - $ generar~~t_{A_{L}} \wedge~~t_{A_1} = t + t_{A_{L}} $  <!-- aqui deberia ser  -->
+  - $ if~(n_1 == 1)~~~then~~~~generar~~t_{A_S}~\wedge ~t_{A_2}=t + t_{A_S})$   
```
tienes un par&eacute;ntesis d + ah&iacute;

maux96 committed:
#9 Arreglo de reporte.md

MakeMake23 reviewed (approved):
Ya, todo en talla :grin:
